events/tests/test_permissions.py

```diff
@@ -84,18 +86,22 @@ def test_can_edit_event(membership_status, expected_public, expected_draft):
         )
 
 
+@pytest.mark.no_use_audit_log
 @pytest.mark.parametrize(
-    "is_admin,is_regular_user,expected",
+    "is_admin,is_registration_admin,is_regular_user,expected",
     [
-        (True, True, False),
-        (True, False, False),
-        (False, True, False),
-        (False, False, True),
+        (True, False, True, False),
+        (True, False, False, False),
+        (False, False, True, False),
+        (True, True, False, False),
+        (False, True, True, False),
+        (False, True, False, False),
+        (False, False, False, True),
```

Comment:
`(True, True, True, False)` combination is missing here. Not a big dealt. Just mentioning if it's not intentional.

Reply:
Very good catch! I fixed it in 56b8b1e55b9e97fc48220fa59e80898c51bacabd.